tests/unit/src/test/resources/mtags-scala3/example/TypeEnum.scala

```diff
@@ -0,0 +1,5 @@
+package example
+
+enum TypeEnum/*example.TypeEnum#*/:
+  case Product/*example.TypeEnum.Product.*//*example.TypeEnum.Product#*/(val id/*example.TypeEnum.Product#id.*/: String)
+  case Coproduct/*example.TypeEnum.Coproduct.*//*example.TypeEnum.Coproduct#*/()
```

Comment:
```suggestion
enum TypeEnum/*example.TypeEnum#*/:
  case Product/*example.TypeEnum.Product#*/(val id/*example.TypeEnum.Product#id.*/: String)
  case Coproduct/*example.TypeEnum.Coproduct#*/()
```
looks like this can only be classes and we can still have an object defined later like:
```
object TypeEnum:
  object Product
```
so we would not be pointing at the right place